proto/gossip.proto

```diff
@@ -44,6 +46,61 @@ message GossipRequest {
 message GossipResponse {
   // Requested delta of server's infostore.
   optional bytes delta = 1;
+  // TODO(thschroeter): replace bytes with GossipInfoStore
+  //optional GossipInfoStore delta = 1;
   // Non-nil means client should retry with this address.
   optional Addr alternate = 2;
 }
+
+message GossipInfoStore {
```

Comment:
@thschroeter rather than calling this GossipInfoStore, create a new .proto file under gossip/ and name the message `InfoStore` - it should completely replace the existing struct (and that pattern should be used for all the other structs that we need to protofy).